popup.js

```diff
@@ -0,0 +1,170 @@
+  ShowProjects += `
+        <div class="work-title1">
+        <div class="hide-card">
+          <div>
+            <h2>${data.name}</h2>
+            <p class="work-description">
+            ${data.description}
+            </p>
+          </div>
+          <ul class="codes">
+            <li class="box">${data.technologies[0]}</li>
+            <li class="box">${data.technologies[1]}</li>
+            <li class="box">${data.technologies[2]}</li>
+          </ul>
+        </div>
+        <div class="button-area">
+          <button id="${data.id}" type="button"  class="button">See Project</button>
+        </div>
+      </div>`;
+  MyProjects.innerHTML = ShowProjects;
+});
+
+const showitem = document.querySelectorAll('.button');
+const openitem = document.querySelector('.Main-Popup');
+const overlay = document.querySelector('#overlay');
+
+showitem.forEach((data) => {
+  data.addEventListener('click', (event) => {
+    const Index = Array.from(showitem).indexOf(event.target);
+    PopupMenu.innerHTML = `
+        <header class="head">
+        <div class="top-items">
+        <h1 class="header-1" id="hd-1">${ProjectDetails[Index].name}</h1>
+        <h1 class="header-2" id="hd-2">${ProjectDetails[Index].name}</h1>
+        <button onclick="closePopup()" class="remove-btn"><span  class="close-Icon">&times;</span></button>
+        </div>
+        <nav class="nav-bar">
+        <ul>
+        <li><a href="">html</a></li>
+        <li><a href="">Bootstrap</a></li>
+        <li><a href="">Ruby on rails</a></li>
+        </ul>
+        </nav>
+       </header>
+       <div class="main-section">
+       <section class="section-mobile" style="background-image: url(${ProjectDetails[Index].featured_image});"></section>
+        <section class="section-1"><img src="${ProjectDetails[Index].featured_image}" width=659 height=448 alt=""></section>
+        
+        <section class="section-2">
+            <div class="description">
+                <p class="details">${ProjectDetails[Index].description}</p>
+                      <p class="details-2">${ProjectDetails[Index].description}</p>
+            </div>
+          
+            
+            <div class="live-section">             
+                <ul>
+                    <li><a href="${ProjectDetails[Index].live_version}">See Live</a>
+                    <img class="icon-image" src="black-social/Export.png" alt="live view icon">
+                        <span class="icon-mobile" id="ic" style="background-image: url(${ProjectDetails[Index].icon1});"></span>
+                        <li><a href="">See Source</a>
+                            <img class="icon-image"  src="black-social/Vector.png" alt="live view icon">
+                            <span class="icon-mobile"   style="background-image: url(${ProjectDetails[Index].icon2});"></span></li>
+                            
+                </ul>                         
+            </div>
+        </section>
+    
+    </div>
```

Comment:
- [ ] Great use of template strings :clap: . I have seen the popup screenshots from your pull request description and on my end they seem not to be the same on my end as it looks on your pull request description. Kindly fix this.

**This is how your desktop version looks like:**

![image](https://user-images.githubusercontent.com/86318284/181674397-6030865b-d64b-4365-abb1-f363c61acf08.png)

It is appearing at the bottom side of the screen while on your end it appears correctly at the middle section of the screen.

**This is how your mobile version looks like on my end:**

![image](https://user-images.githubusercontent.com/86318284/181674647-d0935fba-6456-4472-9f60-7e394e712965.png)

I am unable to see the popup description as it is on my end. Also, the margins and padding seem not to be right. I tried to increase the width from the 375px specified on Gigma to 498px to try to see if it will show correctly but it still shows the same way. Kindly look into this.

**This is the desired mobile popup according to the Figma design:**

![image](https://user-images.githubusercontent.com/86318284/181674938-fd641c21-d9a7-4f43-975c-675c22089702.png)

Reply:
Design updated problem solved.thank you